tests/inverted-index-test.js

```diff
@@ -42,7 +212,7 @@ describe('Search index', () => {
   });
 
   it('should return { "understand":[0,1], "from": [1] } for array ["understand","from"]', () => {
-    const result = { 'book2.json': { 'from': [1], 'understand': [0, 1] }, 'book1.json': { 'from': [-1], 'understand': [-1] } };
+    const result = { 'book2.json': { from: [1], understand: [0, 1] }, 'book1.json': { from: [-1], understand: [-1] } };
     expect(indexObj.searchIndex(indexObj.index, ['understand', 'from'])).toEqual(result);
   });
 
```

Comment:
Apart from testing for a json array, you should also test for an empty json array